Documentation/Components/E-commerce Card/commerce-card.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cards</title>
+    <link rel="stylesheet" href="commerce-card.css">
+</head>
+<body>
+    <div class="wrapper flex-row">
+        <div class="card-midtext">
+            <div class=img-container><img class="prod-img" src="/Documentation/Components/assets/1.jpg"></div> 
+            <p class="overlay-text">MEN</p>
+        </div>
+        <div class="card flex-column">
+            <div class=img-container><img class="prod-img" src="/Documentation/Components/assets/2.jpg"></div> 
+            <div class="text-container flex-column">
+                <p class="prod-title">Men's Bomber Jacket</p>
+                <p class="prod-cost">₹1999</p>
+            </div>
+            <div class="btns">
+                <button class="btn-l prim-acc-btn">Add to cart</button>
+                <button class="btn-l prim-btn">Buy Now</button>
+            </div>
+        </div>
+        <div class="card flex-column">
+            <div class=img-container><img class="prod-img" src="/Documentation/Components/assets/2.jpg"></div> 
+            <div class=badge-container><img class="badge-img" src="/Documentation/Components/assets/like.png"></div>
+            <div class="text-container flex-column">
+                <p class="prod-title">Men's Bomber Jacket</p>
+                <p class="prod-cost">₹1999</p>
+            </div>
+            <div class="btns-full flex-column">
+                <button class="btn-l gray-btn">Move to cart</button>
+            </div>
+        </div>
+        <div class="card card-horizontal flex-row">
+            <div class="img-container"><img class="prod-img" src="/Documentation/Components/assets/3.jpg"></div> 
+            <div class="right-section flex-column">
+            <div class="text-container flex-column">
+                <p class="prod-title">Men's Premium Jacket</p>
+                <div class="prod-costs flex-row">
+                <p class="prod-cost">₹1999</p>
+                <p class="prod-mrp">₹3999</p>
+                </div>
+                <p class="prod-discount">50%</p>
+            </div>
+            <div class="btns-95 flex-column">
```

Comment:
doesn't understand `btns-95 `meaning

Reply:
It was for the width of the button, I have added width to the class name now.